packages/jtex/src/jtex.ts

```diff
@@ -145,6 +150,7 @@ class JTex {
       errorLogFn: errorLogger(this.session),
       warningLogFn: warningLogger(this.session),
     };
+    if (bibliography) frontmatter.bibliography = bibliography;
```

Comment:
should we also clone this object? Probably not a big deal for our stuff, but probably best if we are making changes to a different doc.

Reply:
I think the validation functions clone objects, but I did destructure before changing bib.

Reply:
Nice!